test/functional/tests/volumes/test_cache1_core4_lvm4.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import datetime
+
+import pytest
+
+from api.cas import casadm
+from api.cas.init_config import InitConfig
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from storage_devices.lvm import Lvm, LvmConfiguration
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite, VerifyMethod
+from test_utils.os_utils import sync
+from test_utils.size import Size, Unit
+
+opencas_conf_path = "/etc/opencas/opencas.conf"
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core1", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core2", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("filesystem", Filesystem)
+def test_cache1_core4_lvm4(filesystem):
```

Comment:
I suggest to name this test somehow  to be more meaningful, e.g. test_many_lvms_on_core - current name does not say if LVMs are created on CAS device or CAS device is created on LVM ;)